api/elastic.py

```diff
@@ -462,15 +480,19 @@ def applyFilters(self, size=100):
                 inner_hits=innerHitsClause, query=Q('match_all')
             )
 
+        if self.govDocFilter != None:
+            self.query = self.query.query(self.govDocFilter)
+
     def applyAggregations(self):
         rootAgg = self.query.aggs.bucket(
             'editions', A('nested', path='editions')
         )
 
         lastAgg = rootAgg
         for i, agg in enumerate(self.appliedAggregations):
-            currentAgg = 'edition_filter_{}'.format(i)
-            lastAgg = lastAgg.bucket(currentAgg, agg)
+            if agg != A('filter', **{'term': {'is_government_document': False}}) and agg != A('filter', **{'term': {'is_government_document': True}}):
```

Comment:
A few thoughts, doesn't look like the aggregation object is being constructed above, so this isn't being used right now.

Also I'm not sure that testing for equivalence of `A()` objects will work, depending on the implementation.

I'd get rid of this, and add the aggregation similarly as we did with the filter, as an attribute and then setting down where we create the bucket.